### ISSUE TITLE
grunt depends on .jshintrc, which is not generated

### ISSUE THREAD

**Issue by odigity** [yeoman/generator-webapp_DEPRECATED#163]
Brand new to yeoman.

Just created a new webapp to try it out with 'yo webapp myapp', then ran grunt to see what the build process does.  Got this:

Running "jshint:all" (jshint) task
ERROR: Can't parse config file: .jshintrc

It seems odd that the default build process would depend on a config file that wasn't auto-generated by the webapp generator.  Is the user expected to provide one themselves?  I've never used jshint, so I'm not sure what to put in it yet.

**Comment by stephenplusplus** [comment]
Here are the files which are templated/copied and created when you run `yo webapp`: https://github.com/yeoman/generator-webapp/tree/master/app/templates

Here is the `.jshintrc` file that should be created in your project's directory: https://github.com/yeoman/generator-webapp/blob/master/app/templates/jshintrc

My only guess as to why your project didn't include this file would be there was an error that interrupted the scaffolding process.

Try creating a file `.jshintrc` in the root directory of your project (where `package.json` lives) that contains:

*\* Note: the below snippet is subject to change. For the latest version, see https://github.com/yeoman/generator-webapp/blob/master/app/templates/jshintrc **

``` js
{
    "node": true,
    "browser": true,
    "esnext": true,
    "bitwise": true,
    "camelcase": true,
    "curly": true,
    "eqeqeq": true,
    "immed": true,
    "indent": 4,
    "latedef": true,
    "newcap": true,
    "noarg": true,
    "quotmark": "single",
    "regexp": true,
    "undef": true,
    "unused": true,
    "strict": true,
    "trailing": true,
    "smarttabs": true,
    "jquery": true
}
```

**Comment by stephenplusplus** [comment]
Also, thanks for trying Yeoman :dancer: 

Sorry it's being all weird.

**Comment by odigity** [comment]
Thanks, that worked.

I ran 'yo webapp myapp' again in a new directory to confirm the behavior -- this time I saw it said it created .jshintrc (along with a few other files I was missing in the first one, like .editorconfig).  When it was done, I checked, and the files were there, and grunt worked.

I went back to the first webapp and confirmed those files still weren't there, then scrolled back in my shell's output buffer and confirmed that the 'created .jshintrc' was actually in the output, despite the file being missing.

So, it's a mystery, and wasn't reproduced with the second attempt.  Probably best to ignore unless it happens again to another person.
